kafka/source/pkg/reconciler/controller.go

```diff
@@ -27,7 +27,7 @@ import (
 	sourcescheme "knative.dev/eventing-contrib/kafka/source/pkg/client/clientset/versioned/scheme"
 	kafkaclient "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/client"
 	kafkainformer "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/informers/sources/v1alpha1/kafkasource"
-	"knative.dev/eventing-contrib/pkg/apis/sources/v1alpha1"
```

Comment:
why is that contrib in here ? :thinking:

Reply:
it's not anymore? :)